ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { DashboardRedesignedComponent, HeaderConfigurationService } from 'toolkit';
+import {
+    ParticipantsSearchServiceAgent,
+    EnrollmentStatusType,
+    AnnouncementsServiceAgent,
+    mockComponent,
+    SessionMementoService, UserInvitationServiceAgent
+} from 'ddp-sdk';
+import { RouterTestingModule } from '@angular/router/testing';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+import { Session } from 'inspector';
+
+describe('DashboardRedesignedComponent', () => {
+    let fixture: ComponentFixture<DashboardRedesignedComponent>;
+    let component: DashboardRedesignedComponent;
+    let debugElement: DebugElement;
+    let participantsSearchSpy: jasmine.SpyObj<ParticipantsSearchServiceAgent>;
+    let announcementsSpy: jasmine.SpyObj<AnnouncementsServiceAgent>;
+    let sessionMock: SessionMementoService;
+
+    beforeEach(async() => {
+        participantsSearchSpy = jasmine.createSpyObj('participantsSearchSpy', {
+            getParticipant: of({
+                guid: '1234',
+                hruid: '5678',
+                status: EnrollmentStatusType.REGISTERED,
+            })
+        });
+        announcementsSpy = jasmine.createSpyObj('participantsSearchSpy', {
+            updateSelectedUser: undefined,
+            resetSelectedUser: undefined,
+            getMessages: of([])
+        });
+        sessionMock = { isAuthenticatedAdminSession: () => true, session: {} as Session } as unknown as SessionMementoService;
+        const headerConfigSpy = jasmine.createSpyObj('participantsSearchSpy', ['setupDefaultHeader']);
+        const userInvitationSpy = jasmine.createSpyObj('userInvitationSpy', { getInvitations: of([]) });
+
+        const dashboard = mockComponent({ selector: 'ddp-user-activities', inputs: ['studyGuid', 'selectedUserGuid', 'displayedColumns'] });
+        const subjectPanel = mockComponent({ selector: 'ddp-subject-panel', inputs: ['subject'] });
+        await TestBed.configureTestingModule({
+            imports: [
+                RouterTestingModule,
+                NoopAnimationsModule
```

Comment:
I guess it will work without `NoopAnimationsModule` as well.

Reply:
see my answer above